lib/AST/ASTPrinter.cpp

```diff
@@ -899,7 +908,10 @@ void PrintAST::printPattern(const Pattern *pattern) {
     recordDeclLoc(decl, [&]{
       if (Options.OmitNameOfInaccessibleProperties &&
           contributesToParentTypeStorage(decl) &&
-          !isPublicOrUsableFromInline(decl))
+          !isPublicOrUsableFromInline(decl) &&
+          // FIXME: We need to figure out a way to generate an entry point
+          //        for the initializer expression without revealing the name.
```

Comment:
I don't think that's possible, since the mangled name of the initializer contains the property name

Reply:
We don't actually care what the name is, though, since it's not a public entry point. At least, we don't for resilient modules.

Reply:
If we generate a name here it has to be consistent between invocations though, so that you can do a non-WMO build where multiple source files import the same textual interface (and the cached binary form got rebuild between invocations)

Reply:
I would again advocate for the simpler approach, instead of figuring out the exact strategy for when we can elide the name, just be conservative and only elide it if there's no initializer

Reply:
We're talking about a swiftinterface here; it's always one file.

Reply:
And we can't just print the name because the name might conflict with extensions.